src/doc/src/reference/unstable.md

```diff
@@ -1126,44 +1125,6 @@ You can use the flag like this:
 cargo rustdoc -Z unstable-options --output-format json
 ```
 
-## check-cfg
```

Comment:
Can you add a `## check-cfg` section to the bottom of the file, following the style of the other "stabillized" sections, with a link to the section in build-scripts.md?

Reply:
Sure, missed that part. Done.